experimental/features_example_test.go

```diff
@@ -71,7 +71,7 @@ func ExampleCoreFeaturesThreads() {
 			// among arbitrary goroutine invocations.
 			child := createChildModule(r, mod, wasmCompiled)
 			fn := child.mod.ExportedFunction("run")
-			for i := 0; i < 6000; i++ {
+			for i := 0; i < 100000; i++ {
```

Comment:
I went ahead and raised to a point where interpreter takes about 1s on my machine. What do you think?